tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/eval/ParseBank.scala

```diff
@@ -0,0 +1,65 @@
+package org.allenai.nlpstack.parse.poly.eval
+
+import org.allenai.nlpstack.parse.poly.polyparser._
```

Comment:
Import just what you need-- looks like `PolytreeParse` is the only thing used from this package...